{"uuid":"29cd305c-d3ad-4b01-b840-334f1a9b5799","historyId":"f54b69a2d13396b6750038293700a7ca","fullName":"com.example.teamcity.api.BuildTypeTest.userCreatesTwoBuildTypeTestWithTheSameIdTest","labels":[{"name":"package","value":"com.example.teamcity.api.BuildTypeTest"},{"name":"testClass","value":"com.example.teamcity.api.BuildTypeTest"},{"name":"testMethod","value":"userCreatesTwoBuildTypeTestWithTheSameIdTest"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"teamcity-testing-framework"},{"name":"subSuite","value":"com.example.teamcity.api.BuildTypeTest"},{"name":"host","value":"MBP-Aida.fritz.box"},{"name":"thread","value":"22860@MBP-Aida.fritz.box.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"User should not be able to create 2 build types with the same id","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Can not get Id or locator of an entity","trace":"java.lang.IllegalStateException: Can not get Id or locator of an entity\n\tat com.example.teamcity.api.generators.TestDataStorage.getEntityIdOrLocator(TestDataStorage.java:48)\n\tat com.example.teamcity.api.generators.TestDataStorage.addCreatedEntity(TestDataStorage.java:53)\n\tat com.example.teamcity.api.requests.checked.CheckedBase.create(CheckedBase.java:28)\n\tat com.example.teamcity.api.BuildTypeTest.userCreatesTwoBuildTypeTestWithTheSameIdTest(BuildTypeTest.java:37)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\n\tat org.testng.TestRunner.run(TestRunner.java:614)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\nCaused by: java.lang.NoSuchFieldException: id\n\tat java.base/java.lang.Class.getDeclaredField(Class.java:2782)\n\tat com.example.teamcity.api.generators.TestDataStorage.getEntityIdOrLocator(TestDataStorage.java:36)\n\t... 28 more\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1741557915375,"stop":1741557917243}